runtime/support.py

```diff
@@ -376,7 +376,10 @@ def setup(import_roots, zip_safe, extract_dir=None):
             else:
                 shutil.rmtree(extract_dir, ignore_errors=True)
 
-            extract_dir = os.path.expanduser(extract_dir)
+            # Turn extract_dir into a relative path to join with the extraction root
+            extract_dir = os.path.expanduser(extract_dir).lstrip('/')
```

Comment:
nit: we probably don't care about windows support but `os.path.sep` would technically be more correct here.

the combo of `expanduser` and `PAR_EXTRACT_ROOT` seems kind of strange... if I did
```py
par(
  name = 'app'
  extract_dir = '~/.pars/my_app'
)
```
then ran
```sh
[jbogdanov] $ cd /some/dir
[jbogdanov] $ PAR_EXTRACT_ROOT=. my_app
```
we would extract into `/some/dir/home/jbogdanov/.pars/my_app`.

Although, after writing that out I guess that makes the most sense?

Reply:
Hm, interesting point. It's kind of an odd use case to have a non-absolute (or `~`-based) `extract_dir` anyway, so I wasn't 100% sure what the right behavior here would look like, but I wanted to keep the behavior as close as possible to what it was before.

I think the example you provide makes sense, as opposed to something like `/some/dir/~/.pars/my_app` (literal `~`) or ignoring `PAR_EXTRACT_ROOT` and going to `/home/jbogdanov/.pars/my_app`... so maybe this is the behavior we want?